apps/web/src/components/forms/profile.tsx

```diff
@@ -122,7 +121,10 @@ export const ProfileForm = ({ className, user }: ProfileFormProps) => {
                 <FormControl>
                   <SignaturePad
                     className="h-44 w-full"
-                    containerClassName="rounded-lg border bg-background"
+                    containerClassName={cn(
+                      'rounded-lg border bg-background',
+                      isSubmitting ? 'pointer-events-none opacity-50' : null,
```

Comment:
This can just be `isSubmitting && 'pointer-events-none opacity-50` 😄

Reply:
IMO we should pass a `disabled` prop to the `SignatuePad` which handles it in there for reusability purposes instead of just having it for the form profile

Similar to how it was done before in this pr

Reply:
@Mythie Using `cn` makes the code cleaner and more readable I guess. 
@dguyen I agreed tho, @Mythie Should we have both of the methods integrated? What do you suggest?

Reply:
What do you mean both methods? Wouldn't there only be one?

Reply:
I meant, reducing the opacity and disabling the pointer events vis CSS and programmatically disabling javascript events to be on the safer side.

Reply:
It should be fine to just do it via CSS since this isn't really critical

Reply:
sure, then I belive this PR looks good to me.